Rakefile

```diff
@@ -128,7 +136,7 @@ namespace :acceptance do
       # then the script will remove the old instance before replacing it with the new
       # one.
       puts("Starting the mock ServiceNow instance at the master (#{master.uri})")
-      master.bolt_upload_file('./spec/support/acceptance/servicenow', '/tmp/servicenow')
+      master.bolt_upload_file('./spec/support/docker/servicenow', '/tmp/servicenow')
```

Comment:
Hmm, I'd still like acceptance test stuff to be in `spec/support/acceptance` b/c that's what all this Docker stuff's for. Otherwise, this structure makes it seem like `docker` stuff is also being used for the unit tests when it isn't.

Reply:
`spec/support/acceptance/docker` is fine.